nfc/libs/NfcCoreLib/lib/Nci/phNciNfc.c

```diff
@@ -208,18 +340,30 @@ phNciNfc_Initialise(
             wStatus =  phNciNfc_LogConnInit(pNciContext);
             if(NFCSTATUS_SUCCESS == wStatus)
             {
-                pNciContext->ResetInfo.ResetTypeReq = eResetType;
-                /* Init payload to display the build number in the init response */
-                pNciContext->tInitInfo.bExtension = 0x00;
-                pNciContext->tInitInfo.bSkipRegisterAllNtfs = 0x00;
-                wStatus = phNciNfc_GenericSequence((void *)pNciContext, NULL, NFCSTATUS_SUCCESS);
-                if(NFCSTATUS_PENDING != wStatus)
+                wStatus = phNciNfc_RegAllNtfs(pNciContext);
```

Comment:
Thanks for pointing out. This function got moved firstly in order to be sure we were not going to miss the first NCI_CORE_RESET_NTF after power on or HW reset. 

Moving the phNciNfc_RegAllNtfs call to phNciNfc_Initialize help to ensure that notification callback are registered before the first sequence execution.

If moving back phNciNfc_RegAllNtfs to phNciNfc_InitReset which is in gphNciNfc_Init sequence will cause phNciNfc_Reinitialize to re-execute phNciNfc_RegAllNtfs and trigger the need back for bSkipRegisterAllNtfs.

Also i believe, phNciNfc_Init.c contains NCI sequence for the NFCC initialization.